packages/app-project/pages/projects/[owner]/[project]/about/research.js

```diff
@@ -0,0 +1,55 @@
+import getDefaultPageProps from '@helpers/getDefaultPageProps'
+export { default } from '@screens/ProjectAboutPage'
+
+// export async function getServerSideProps({ params, query, req, res }) {
+//   const { props } = await getDefaultPageProps({ params, query, req, res })
+
+//   return {
+//     props: {
+//       ...props,
+//       pageType: 'science_case'
+//     }
+//   }
+// }
+
+/////////////////
+
+import initStore from '@stores'
+import { getSnapshot } from 'mobx-state-tree'
+
+export async function getStaticPaths() {
+  // dynamic route: /projects/[owner]/[project]/about/research
+  // need to fetch every zooniverse [owner] & [project] here??
+
+  return {
+    paths: [
+      { params: { owner: 'brooke', project: 'i-fancy-cats' } },
+      { params: { owner: 'goplayoutside3', project: 'test-bike-lane-uprising' } }
+    ],
+    fallback: true,
+  }
+}
```

Comment:
@eatyourgreens I tried out the Next.js methods `getStaticPaths` and `getStaticProps` when fetching data for the project about pages. All of the about pages are dynamic routes, so with this Next.js pattern we would need to fetch every single `/owner/project/` slug on line 21 (to pass as params in the `paths` array). 

Do you recommend continuing on this route? Or sticking with `getServerSideProps`?

Reply:
[`fallback:true`](https://nextjs.org/docs/basic-features/data-fetching#fallback-true) should tell NextJS to build those pages at run time, without specifying the paths at build time. I guess the fallback page doesn't automatically refresh? Next 10 introduced [`fallback: blocking`](https://nextjs.org/blog/next-10#blocking-fallback-for-getstaticpaths), which would build a dynamic page on first request, then serve the static build thereafter. We could upgrade to 10 and use that. (see #1972.)

Reply:
Reading those Next docs, it sounds like the fallback page (in Next 9) should display a loading indicator, then automatically refresh once the page is built. I haven't tried that in practice, though.

Reply:
It sounds like what we want here  is `fallback: blocking`, yeah?  Perhaps it's best to do this refactor after the upgrade to v10

Reply:
I wouldn't spend a lot of time on this right now, so definitely go with `getServerSideProps` for development and staging. It is something we'll need to fix at some point, so that we aren't paying for unnecessary API queries and page builds in production.

I've got the PH-TESS home page building locally with this page setup:
```js
export async function getStaticProps({ params, query = {}, req = {}, res = {} }) {
  const { props } = await getDefaultPageProps({ params, query, req, res })
  return ({ props })
}

export function getStaticPaths() {
  return {
    paths: [],
    fallback: true
  }
}
```

However, setting this up I discovered that the page `<head>` gets the host (for preview links) from request headers. We'll need to fix that before we can use static page generation for production projects.

Reply:
Here's my quick change to build a static project home page for any production project, cached for 300s. The secret seems to be checking `router.isFallback` to see if the props have been generated yet. The page does automatically refresh once the props are ready, which is neat. https://github.com/zooniverse/front-end-monorepo/commit/1a6efd9e3eebdfad57351438c2486aef774ef673